libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -59,18 +46,26 @@ export class UserProfileService {
         nationalId: user.nationalId,
       })
 
-      const islyklarData = await this.getIslyklarData(user.nationalId)
+      const islyklarData = await this.islyklarService.getIslykillSettings(
```

Comment:
During the last meeting we discussed that it would be useful to add this logic to the UserProfile service so other “User Profile” consumers don’t need to integrate directly with the temporary Islykill service.

I guess it’s not an issue for the Application System, which currently requests the user profile through the GraphQL API (and gets this logic). But we also need this in the IDP so it can put these claims into ID tokens and userinfo responses. 

Probably not the end of the world to integrate it directly in that one place. Not a blocker for this PR at least.